smg-automotive/components-pkg
# fix: popover component

[ST-444](https://smg-au.atlassian.net/browse/ST-444?atlOrigin=eyJpIjoiODFhZWI0NjNlNDAzNDAxNjhkYzU1MzkwY2IwZTkwZmEiLCJwIjoiaiJ9)

## Motivation and context

There is a white space at the bottom of cockpit. 
It turns out that this issue is related to Popover component. It's also visible in TopVehicle and TopListing page. I could not find any cases where we use the Popover component in listings-web or any other projects.
This PR fixes the issue without impacting the behavior of the component. [Here](https://github.com/smg-automotive/seller-web/pull/2698) is the PR for seller-web.

## Before

White space at the bottom of Cockpit page (but also TopVehicle and Toplisting pages)
<img width="1150" alt="Screenshot 2024-12-13 at 08 50 12" src="https://github.com/user-attachments/assets/bbd19949-da4f-49e1-93b4-35c9ef51d5e2" />

## After

No more white space

## How to test

Please make sure the Popover behaves as before (all Tooltip + Vehicle filters in the Cockpit page and some tooltips in TopVehicle and TopListing pages) 
<img width="437" alt="Screenshot 2024-12-13 at 08 41 35" src="https://github.com/user-attachments/assets/b0eb8b92-e321-4397-9373-f2859631d59d" />
<img width="350" alt="Screenshot 2024-12-13 at 08 41 51" src="https://github.com/user-attachments/assets/55d04e21-ceec-4b3d-891a-6ac4602c019e" />
<img width="400" alt="Screenshot 2024-12-13 at 08 42 12" src="https://github.com/user-attachments/assets/f55b89fb-04d1-4286-83eb-092eefdb8101" />
<img width="954" alt="Screenshot 2024-12-13 at 08 42 22" src="https://github.com/user-attachments/assets/225265ef-aac5-45f8-af22-2f15ee0dd474" />
<img width="494" alt="Screenshot 2024-12-13 at 08 42 28" src="https://github.com/user-attachments/assets/845ea0c3-4b5f-45c6-b1c0-35a1525de0ea" />

Thank you 🐞 


[ST-444]: https://smg-au.atlassian.net/browse/ST-444?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
